lib/scrivener/paginater/ecto/query.ex

```diff
@@ -22,14 +22,14 @@ defimpl Scrivener.Paginater, for: Ecto.Query do
     %Page{
       page_size: page_size,
       page_number: page_number,
-      entries: entries(query, repo, page_number, page_size, caller),
+      entries: entries(query, repo, page_number, page_size, caller, options),
       total_entries: total_entries,
       total_pages: total_pages
     }
   end
 
-  defp entries(query, repo, page_number, page_size, caller) do
-    offset = page_size * (page_number - 1)
+  defp entries(query, repo, page_number, page_size, caller, options \\ []) do
+    offset = if options[:offset] != nil, do: options[:offset], else: page_size * (page_number - 1)
```

Comment:
We should use https://hexdocs.pm/elixir/Keyword.html#get_lazy/3 here.

```suggestion
    offset = Keyword.get_lazy(options, :offset, fn -> page_size * (page_number - 1) end)
```